src/components/order-service/order-service.html

```diff
@@ -75,7 +75,7 @@
         type="button"
         class="btn btn-primary wizard-pf-next"
         ng-if="$ctrl.currentStep.id === 'configure'"
-        ng-disabled="!$ctrl.selectedProject || $ctrl.currentStep.id === 'results'"
+        ng-disabled="!$ctrl.selectedProject || $ctrl.currentStep.id === 'results' || $ctrl.invalidNewProject()"
```

Comment:
Here's an example of how you can use ng-form:

https://github.com/openshift/origin-web-console/blob/95b0902ecb3627301fedf2a0a6855cbd5e8c39a2/app/views/directives/select-containers.html

Then we embedded the `select-container` directive in a larger form, which was marked invalid if a container wasn't selected.